demo/simple-column-sorting-pipeline.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
@rdepena As an outsider, can you comment if this makes sense. You only need to review this file.